modules/reindex/src/main/java/org/opensearch/index/reindex/remote/RemoteScrollableHitSource.java

```diff
@@ -186,8 +186,9 @@ public void onSuccess(org.opensearch.client.Response response) {
                             }
                             if (xContentType == null) {
                                 try {
+                                    logger.debug("Response didn't include Content-Type: " + bodyMessage(response.getEntity()));
                                     throw new OpenSearchException(
-                                        "Response didn't include Content-Type: " + bodyMessage(response.getEntity()));
+                                        "Response didn't include supported Content-Type, remote is likely not an OpenSearch instance");
```

Comment:
Why this explicit change in messaging?

Reply:
This is required to reducing the verbosity of error messages in order to exclude the content of a failed request.
For example, if a non OpenSearch URL is provided for remote, the response contains
the entire HTML content of the page. This needs to be removed from the response.